salesforce/10.10/modules/ROOT/pages/salesforce-connector-processing-events.adoc

```diff
@@ -9,88 +9,88 @@ You can use Anypoint Connector for Salesforce (Salesforce Connector) as an inbou
 * *Replay channel listener*
 
 [NOTE]
-When deploying apps on a distributed environment (such as a runtime cluster), the source operation must run on the primary node. If the same source runs on multiple nodes, it causes duplicate event consumption.
+When deploying apps on a distributed environment such as a runtime cluster, the source operation must run on the primary node. If the same source runs on multiple nodes, it causes the consumption of duplicate events.
+
+Each event that travels through your flow contains information about the Salesforce data that changed, including how the data changed and when the change occurred.
+
+Salesforce stores events for 24 hours (or 72 hours for high-volume events). A subscriber to a topic or channel can retrieve events related to that topic or channel during the 24-hour retention window. After the retention window ends, the subscriber can retrieve newer events that have not yet expired.
+
+Salesforce assigns each broadcast event a numeric ID. IDs are incremented, but not necessarily by 1 for each consecutive event. For example, the event following the event with ID `999` can have an ID of `1025`. A broadcast event ID is unique for the organization and channel. Salesforce does not reuse the IDs of deleted events.
 
 == Prerequisites
 
-To process events with Salesforce Connector, ensure you meet the following prerequisites:
+To process events with Salesforce Connector, ensure that you have:
+
+* Access to the Salesforce target resource
+* The required Salesforce Streaming API permissions enabled in your organization
 
-* You must have access to the Salesforce target resource.
-* Before you can receive events for Salesforce changes associated with a topic, you must create the topic, if it does not exist.
-* You must have the required Salesforce Streaming API permissions enabled in your organization.
+Before you can receive events for Salesforce changes associated with a topic, create a topic using Salesforce Connector if the topic doesn't already exist.
 
 [[objectstoreusage]]
-== Object Store Usage
+== Object Store Usage in Events
 
-Both Salesforce Connector and Mule use an object store to persist data for features such as automatic message replay and message redelivery:
+Both Salesforce Connector and Mule runtime engine (Mule) use an object store to persist data for features such as automatic message replay and message redelivery:
```

Comment:
We should only mention that the Salesforce Connector uses an object store to persist data for feature such as...

Reply:
No @stefangretcan it is important to mention that the runtime uses OS too because if there are only 10 transactions per second allowed, then there might be other components also consuming some of those transactions.